src/java.base/share/native/libjli/java.c

```diff
@@ -806,8 +806,6 @@ CheckJvmType(int *pargc, char ***argv, jboolean speculative) {
 static void
 SetJvmEnvironment(int argc, char **argv) {
```

Comment:
This doesn't seem to do anything any more.

Reply:
Thanks for this hint. This brought me to JDK-8186483, which can probably get rolled back now entirely.